Week3/Homework-Week3/assets/morejs_3.js

```diff
@@ -0,0 +1,14 @@
+{
+  /* Create an object and a function that takes the object 
+   as a parameter and prints out all of its properties and values.
+  */
+
+  const car = { made: 'Honada', year: 2019, color: 'Red', availableNow: true };
+
+  function myFunc() {
```

Comment:
`and a function that takes the object as a parameter and prints out all of its properties and values.`

It's almost correct, but the function itself should take an `object` as a parameter and print it